docs/index.rst

```diff
@@ -11,6 +11,11 @@ If this is your first time learning about Dockstore, we recommend starting with
 you to the core concepts of Dockstore, leaving you with a good understanding of the platform. However, if you are simply looking to launch tools and workflows, we recommend
 going straight to the :doc:`End User Topics <end-user-topics/end-user-topics>` or our `quickstart guide <https://dockstore.org/quick-start>`_.
 
+.. toctree::
+   :maxdepth: 1
+
+   Return to Dockstore Home <https://dockstore.org>
```

Comment:
I don't really like "Return to". Nor "Home" :) . Return, because you may not have started here. I would say just say `Dockstore.org` or `Go to Dockstore`.